bundles/org.openhab.binding.lghombot/README.md

```diff
@@ -0,0 +1,95 @@
+# LG HomBot Binding
```

Comment:
I believe I would like a review from the @openhab/documentation-maintainers here if possible. Thanks!